hack/test-cover.sh

```diff
@@ -24,8 +24,8 @@ COVERPROFILE_TMP="$(dirname $0)/../test.coverprofile.tmp"
 COVERPROFILE_HTML="$(dirname $0)/../test.coverage.html"
 
 echo "mode: set" > "$COVERPROFILE_TMP"
-find . -name "*.coverprofile" -type f | xargs cat | grep -v mode: | sort -r | awk '{if($$1 != last) {print $$0;last=$$1}}' >> "$COVERPROFILE_TMP"
-cat "$COVERPROFILE_TMP" | grep -vE "\.pb\.go|\/test\/|zz_generated" > "$COVERPROFILE"
+find . -name "*.coverprofile" -type f | xargs cat | grep -v mode: | sort -r | awk '{if($1 != last) {print $0;last=$1}}' >> "$COVERPROFILE_TMP"
+cat "$COVERPROFILE_TMP" | grep -vE "\.pb\.go|zz_generated" > "$COVERPROFILE"
```

Comment:
Why do we include `test` now?

Reply:
I assume, we don't want to exclude packages that contain `test` in their path, e.g. `/pkg/utils/test`.
`/test` is anyways not run in `make {test,test-cov}`, so I think, it isn't needed anymore.